src/SamplesApp/SamplesApp.net6mobile/SamplesApp.net6mobile.csproj

```diff
@@ -94,12 +94,12 @@
 
 
 	<ItemGroup Condition="'$(TargetFramework)'=='net6.0-android'">
-		<PackageReference Include="Xamarin.Google.Android.Material" Version="1.4.0.4" />
-		<PackageReference Include="Uno.UniversalImageLoader" Version="1.9.36" />
+		<PackageReference Include="Xamarin.Google.Android.Material" />
```

Comment:
Let's keep the android ones in place, particularly because they're different per target framework. (or let's change it in the directory.build.props perf target framework version)

Reply:
@jeromelaban All `Xamarin.Google.Android.Material` are referenced with the same version